lib/Models/CkanCatalogGroup.js

```diff
@@ -329,8 +329,7 @@ CkanCatalogGroup.prototype._load = function() {
 
     var promises = [];
     for (var i = 0; i < this.filterQuery.length; i++) {
-        var url = cleanAndProxyUrl(this, this.url) + '/api/3/action/package_search?rows=100000&' + this.filterQuery[i];
-
+        var url = cleanAndProxyUrl( this.terria, this.url) + 'api/3/action/package_search?rows=100000&' + this.filterQuery[i];
```

Comment:
Thanks, yep. The change from `/api` to `api/` is intentional - totally unrelated to this branch, but necessary (vanilla CKANs don't like foo//api URLs).